src/commands/synthetics/run-tests-lib.ts

```diff
@@ -192,52 +168,117 @@ const getTestListBySearchQuery = async (
   }))
 }
 
-export const getTestsList = async (
+export const getFinalVersionOfTestsFromCi = async (
```

Comment:
nit, I would suggest to use the name we are going to use in the config files: `test config override`, what do you think?
```suggestion
export const getTestConfigOverrideList = async (
```